snippets/deployment-config-deprecated.adoc

```diff
@@ -0,0 +1,18 @@
+// Text snippet included in the following assemblies:
+//
+// * applications/deployments/what-deployments-are.adoc
+// * applications/deployments/managing-deployment-processes.adoc
+//
+// Text snippet included in the following modules:
+//
+// * modules/deployments-deploymentconfigs.adoc
+// * modules/deployments-comparing-deploymentconfigs.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+As of {product-title} 4.14, `DeploymentConfig` objects are deprecated. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
```

Comment:
It is important to avoid mentioning removal of DeploymentConfig from future releases. So something like `however, only security and critical fixes are accepted. It is recommended to migrate to Deployments where applicable.` or similar.